### PR TITLE
MKM 11 cards

### PR DESCRIPTION
**Pull request by Simisays**
I think the last cards that don't require engine changes ?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+A:SP$ DestroyAll | ValidCards$ Creature | SubAbility$ DBExile | SpellDescription$ Destroy all creatures.
+SVar:DBExile:DB$ ChangeZone | ChangeType$ Card.OppCtrl | Hidden$ True | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | ConditionDefined$ Collected | ConditionPresent$ Card | SubAbility$ ExileYard | SpellDescription$ If evidence was collected, exile a card from an opponent's graveyard. Then search its owner's graveyard, hand, and library for any number of cards with that name and exile them. 
+SVar:ExileYard:DB$ ChangeZone | ChangeType$ Remembered.sameName | Origin$ Graveyard | DefinedPlayer$ RememberedController | Chooser$ You | ConditionDefined$ Collected | ConditionPresent$ Card | Destination$ Exile | ChangeNum$ NumInYard | Hidden$ True | SubAbility$ ExileHand 
+SVar:ExileHand:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | DefinedPlayer$ RememberedController | ChangeType$ Remembered.sameName | ConditionDefined$ Collected | ConditionPresent$ Card | ChangeNum$ NumInHand | Chooser$ You | SubAbility$ ExileLib | StackDescription$ None
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt:8]: Don't think `RememberedController `is correct here ? It doesn't seem to work after the chosing of card in opponents grave

File: forge-gui/res/cardsfolder/upcoming/push_pull.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+DeckHas:Ability$Graveyard|Sacrifice 
+Oracle:Destroy target tapped creature.
+
+ALTERNATE
+
+Name:Pull
+ManaCost:4 BR BR
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TargetMin$ 0 | TargetMax$ 2 | TargetsFromSingleZone$ True | ValidTgts$ Creature | TgtPrompt$ Select up to two target creature cards from a single graveyard | GainControl$ True | SubAbility$ DBPump | SpellDescription$ Put up to two target creature cards from a single graveyard onto the battlefield under your control. They gain haste until end of turn. Sacrifice them at the beginning of the next end step.
+SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Haste | AtEOT$ Sacrifice
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/push_pull.txt:15]: Should this be an animate subability ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/push_pull.txt:15]: nooo

File: forge-gui/res/cardsfolder/upcoming/projektor_inspector.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Projektor Inspector
+ManaCost:2 U 
+Types:Creature Human Detective
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Detective.YouCtrl+Other,Card.Self | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME or another Detective enters the battlefield under your control and whenever a Detective you control is turned face up, you may draw a card. If you do, discard a card.
+T:Mode$ TurnFaceUp | ValidCard$ Detective.YouCtrl | Execute$ TrigDraw | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME or another Detective enters the battlefield under your control and whenever a Detective you control is turned face up, you may draw a card. If you do, discard a card.
+SVar:TrigDraw:AB$ Draw | Cost$ Discard<1/Card> | NumCards$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/projektor_inspector.txt:7]: wrong order

File: forge-gui/res/cardsfolder/upcoming/sudden_setback.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Name:Sudden Setback
+ManaCost:2 U U 
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | AlternativeDecider$ TargetedOwner | Origin$ Battlefield | Destination$ Library | DestinationAlternative$ Library | LibraryPositionAlternative$ -1 | StackDescription$ {p:TargetedOwner} puts {c:Targeted} on the top or bottom of their library. | SpellDescription$ The owner of target nonland permanent puts it on the top or bottom of their library.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sudden_setback.txt:4]: incomplete

File: forge-gui/res/cardsfolder/upcoming/suspicious_detonation.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Suspicious Detonation
+ManaCost:4 R
+Types:Sorcery
+S:Mode$ ReduceCost | ValidCard$ Card.Self | CheckSVar$ X | Type$ Spell | Amount$ 3 | EffectZone$ All | Description$ This spell costs {3} less to cast if you've sacrificed an artifact this turn.
+SVar:X:PlayerCount$SacrificedThisTurn Artifact
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/suspicious_detonation.txt:5]: wrong count

File: forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Name:Vitu-Ghazi Inspector
+ManaCost:1 G
+Types:Creature Elf Detective
+PT:1/3
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. (Exile cards with total mana value 6 or greater from your graveyard.)
+K:Reach
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | CheckSVar$ Collected | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ When CARDNAME enters the battlefield, if evidence was collected, put a +1/+1 counter on target creature and you gain 2 life.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt:7]: did you check the Condition works?

**Comment by Hanmac** [comment]
@Simisays the other 4 cards should be scriptable too?

https://github.com/orgs/Card-Forge/projects/43/views/1

* Airtight Alibi: "Can't be suspected" i added a Static Ability for this
* Hide in Plain Sight: this is just a Cloak of Multiple 
* Bite Down on Crime: @tehdiplomat  this should be possible, right?
* Case of the Burning Masks: this should use the Damage History? @tool4ever can you check this?

**Comment by tool4ever** [comment]
for the Case something like this should already work:
`Count$NumDamageThisTurn Card.YouCtrl,Emblem.YouCtrl Player,Permanent`

File: forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt (hunk 6 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt:7]: Hmm it's not working. I don't think I'm understanding how the collected condition works ....

File: forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
+Name:Deadly Cover-Up
+ManaCost:3 B B
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. (Exile cards with total mana value 6 or greater from your graveyard.)
+A:SP$ DestroyAll | ValidCards$ Creature | SubAbility$ DBExile | SpellDescription$ Destroy all creatures.
+SVar:DBExile:DB$ ChangeZone | ChangeType$ Card.OppCtrl | Hidden$ True | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | ConditionDefined$ Collected | ConditionPresent$ Card | SubAbility$ ExileYard | SpellDescription$ If evidence was collected, exile a card from an opponent's graveyard. Then search its owner's graveyard, hand, and library for any number of cards with that name and exile them.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt:6]: Mandatory$

File: forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt:8]: you messed up the ChangeNum SVar

File: forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt [hunk 8]
```diff
@@ -0,0 +1,10 @@
+Name:Bite Down on Crime
+ManaCost:3 G
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. This spell costs {2} less to cast if evidence was collected. (To collect evidence 6, exile cards with total mana value 6 or greater from your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:4]: fix double description

File: forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | Relative$ True | CheckSVar$ Z | Description$ This spell costs {2} less to cast if evidence was collected. (To collect evidence 6, exile cards with total mana value 6 or greater from your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:5]: are you aware it doesn't work?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:5]: Yeah, I tried different ways to check if collect evidence was paid. Any suggestions what would work here ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:5]: Try `Count$OptionalGenericCostPaid.2.0`

it's not perfect but probably ok

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:5]: Thanks, working now !

File: forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt (hunk 6 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt]: I've used the same method like you showed below. It's working now

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt]: no, that's not how the card I've mentioned does it

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt]: `Hmm ConditionDefined$ Collected | ConditionPresent$ Card  isn't working `

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt]: well the script has `ConditionDefined$ CastSA>Collected` so how about using that 🦀

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt]: Yeah I tried that first but that didn't work

File: forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt [hunk 10]
```diff
@@ -0,0 +1,10 @@
+Name:Bite Down on Crime
+ManaCost:3 G
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6.
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ Z | EffectZone$ All | Relative$ True | Description$ This spell costs {2} less to cast if evidence was collected. (To collect evidence 6, exile cards with total mana value 6 or greater from your graveyard.)
+SVar:Z:Count$OptionalGenericCostPaid.2.0
+A:SP$ Pump | AILogic$ Fight | NumAtt$ +2 | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | SubAbility$ DBFight | SpellDescription$ Target creature you control gets +2/+0 until end of turn. It deals damage equal to its power to target creature you don't control.
+SVar:DBFight:DB$ Fight | Defined$ ParentTarget | ValidTgts$ Creature.YouDontCtrl | AILogic$ Always | TgtPrompt$ Select target creature you don't control
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt:8]: why is this Fight lol?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**